Orange/widgets/visualize/owboxplot.py

```diff
@@ -738,6 +742,16 @@ def stat_ANOVA():
                 t = "" if np.isnan(F) else f"ANOVA: {F:.3f} (p={p:.3f}, N={n})"
         self.stat_test = t
 
+    def _compute_tests_disc(self):
+        if self.group_var is None or self.attribute is None:
+            self.stat_test = ""
+        else:
+            chi, p, dof = self._chi_square(self.group_var, self.attribute)
+            if np.isnan(p):
+                self.stat_test = ""
+            else:
+                self.stat_test = f"χ²: {chi:.2f} (p={p:.3f}, dof={dof})"
```

Comment:
I like that this is shown below, but after clicking 'Stretch bars', it disappears and never shows up again (unless attribute is changed).